scss/chat/_chat-list-item.scss

```
@@ -75,6 +77,14 @@
         color: var(--chat-list-item-summary-text);
         user-select: none;
 
+        & > img.summary_thumbnail {
+          height: 18px;
+          width: 18px;
```

```suggestion
          height: 18px;
          width: 18px;
          min-width: 18px;
          min-height: 18px;
```

adding these two helps that the thumbnails always have the same size.

But maybe we want to switch to background image and use background-cover and size to get a less stretched result, maybe a bit zoomed in.

https://developer.mozilla.org/en-US/docs/Web/CSS/background